### DISCUSSION TITLE
darkerMode and others works weird

### DISCUSSION THREAD

**Discussion by NukeThief** [PoomSmart/YTABConfig#20] [Q&A] [answered]
Some hud elements are still on the old dark mode version.

My settings are:

`Total: 14
YTColdConfig.enableCinematicContainer: 1
YTColdConfig.enableDarkerDarkMode: 1
YTColdConfig.enableSwipeToRemoveInPlaylistWatchEp: 1
YTColdConfig.removeNextPaddleForSingletonVideos: 1
YTColdConfig.removePreviousPaddleForSingletonVideos: 1
YTColdConfig.replaceNextPaddleWithFastForwardButtonForSingletonVods: 1
YTColdConfig.replacePreviousPaddleWithRewindButtonForSingletonVods: 1
YTColdConfig.uiSystemsClientGlobalConfigEnableModernButtonsForNativeLongTail: 1
YTColdConfig.uiSystemsClientGlobalConfigEnableRoundedDialogForNative: 1
YTColdConfig.useDarkerPaletteBgColorForElements: 1
YTColdConfig.useDarkerPaletteTextColorForElements: 1
YTHotConfig.enableLegacyBehaviorForFailedIdentityLookupsInEventLoggingService: 0
YTHotConfig.enableLinearLiveUxIos: 1
YTHotConfig.enableLocationResolutionUxIos: 1`
![image](https://user-images.githubusercontent.com/81801547/198280013-7bab8303-c5d8-4e41-947d-8d2978b1b808.jpeg)

**Comment by PoomSmart** [comment]
See https://github.com/PoomSmart/YTDarker/blob/main/Tweak.x for possible reasons.

**Comment by NukeThief** [comment]
do i have to do it one by one or can i import it someway? (i use uYouPlus)

**Reply by PoomSmart** [reply]
uYouPlus doesn't integrate YTDarker tweak at the moment due to incompatibility with uYou.

**Reply by NukeThief** [reply]
ohhh

**Reply by NukeThief** [reply]
Using uYouPlus's OLED tweak seems to be the workaround for now.![image](https://user-images.githubusercontent.com/81801547/198282067-a336dd85-761e-4ae6-a284-e75fd130e76f.jpeg)

**Comment by NukeThief** [comment, marked as answer]
The uYou team and contributors are currently working on fixing this issue.
